README.md

```diff
@@ -17,6 +17,8 @@ Hier zwei Stück genannt, welchen man beim Einsatz eines Open-Source Java-Projek
 * [OpenJDK-Webseite](https://openjdk.org/) und [Wikipedia-Eintrag](https://de.wikipedia.org/wiki/OpenJDK)
 * [Amazon Corretto](https://aws.amazon.com/de/corretto/)
 
+Für Java wollen wir eine einheitliche Codeformatierung. Diese wird sichergestellt durch die Nutzung von [Spotless](https://github.com/diffplug/spotless/tree/main/plugin-maven) Codeformatter in Kombination mit [it@M Codeformat](https://github.com/it-at-m/itm-java-codeformat). Als Linter wird [Checkstyle](https://checkstyle.org/) genutzt.
```

Comment:
> Als Linter wird [Checkstyle](https://checkstyle.org/) genutzt.

ich kenne kein OSS Projekt von it@M wo tatsächlich direkt Checkstyle im CI integriert ist. und auch bei internen Projekten ist mir da keines bekannt. 

Den Satz würde ich daher weglassen, es sei denn ich bin hier nicht richtig informiert ;-)

Reply:
Ich habe beim Umsetzen von Spotless auch Checkstyle eingeführt und der Check wird in der GitLab Pipeline und beim Maven Build ausgeführt. Heißt, Checkstyle dient als Linter dann in den zukünftigen Projekten. Wenn du trotzdem der Meinung bist, dass Checkstyle nicht in den Info-Text reingehört, kann ich es gern löschen.